contrib/guix/manifest.scm

```diff
@@ -577,8 +558,9 @@ inspecting signatures in Mach-O binaries.")
                (base32
                 "1b2n1gxv9f4fd5yy68qjbnarhf8mf4vmlxk10i3328c1w5pmp0ca"))
               (patches (search-our-patches "glibc-ldd-x86_64.patch"
+                                           "glibc-versioned-locpath.patch"
                                            "glibc-2.27-riscv64-Use-__has_include-to-include-asm-syscalls.h.patch"
-                                           "glibc-2.27-dont-redefine-nss-database.patch"
```

Comment:
Why dropping the `glibc-2.27-dont-redefine-nss-database.patch` is safe?

Reply:
The patch is included in the branch. I've also bumped the branch to be the head of the current 2.27 release branch, which contains some additional backported fixes.